en,zh: Adjust for v1 CRD (#1434)
pingcap/docs-tidb-operator: en/deploy-tidb-from-kubernetes-gke.md

TiDB Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.

```shell
kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
kubectl get crd tidbclusters.pingcap.com
```

> **Note:**
>

> > For Kubernetes < 1.16, it only support v1beta1 CRD, so we have to change the file "crd.yaml" in above commands to "crd_v1beta1.yaml".

Reviewer: ```suggestion
> For Kubernetes earlier than 1.16, only v1beta1 CRD is supported, so you need to change `crd.yaml` in the above command to `crd_v1beta1.yaml`.
```
